pax-runtime/src/repeat.rs

```diff
@@ -159,7 +143,15 @@ impl InstanceNode for RepeatInstance {
                         panic!("ran evaluator after expanded node dropped (repeat elem)")
                     };
                     let source = source_expression.get();
-                    let source_len = source.len();
+                    let source_len = if let PaxValue::Range(start, end) = source {
```

Comment:
Nothing we need to do now, but would be nice if this could be a try_coerce impl for Vec<T> that coerces a PaxValue::Range into a Vec. I remember you mentioned it was a bit nasty because of the trait bounds?

Reply:
would also potentially allow us to iterate over object key/values by impl try_coerce for Vec<(String, B)> for PaxValue::Object

Reply:
yeah I tried making that work but the trait bounds are nasty for Vec<T> since there is no way to coerce a generic range<T> into a Vec<T>.